lacework/resource_lacework_alert_channel_splunk.go

```diff
@@ -0,0 +1,267 @@
+package lacework
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
```

Comment:
nitpick: separate internal and external imports
```suggestion
	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"

```